<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="sbasak22" name="com.training.sanity.tests.ChangeRoleOfUsers" tests="3" failures="0" timestamp="09 Nov 2019 15:39:25 GMT" time="10.881" errors="1">
  <testcase name="setUp" time="10.881" classname="com.training.sanity.tests.ChangeRoleOfUsers">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.training.sanity.tests.ChangeRoleOfUsers.setUp(ChangeRoleOfUsers.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase name="changeRoleOfUsers" time="0.000" classname="com.training.sanity.tests.ChangeRoleOfUsers">
    <skipped/>
  </testcase> <!-- changeRoleOfUsers -->
  <testcase name="changeRoleOfUsers" time="0.000" classname="com.training.sanity.tests.ChangeRoleOfUsers">
    <skipped/>
  </testcase> <!-- changeRoleOfUsers -->
</testsuite> <!-- com.training.sanity.tests.ChangeRoleOfUsers -->
